ee/agent/storage/sqlite/logstore_sqlite.go

```diff
@@ -0,0 +1,100 @@
+package agentsqlite
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+func (s *sqliteStore) AppendValue(timestamp int64, value []byte) error {
+	colInfo := s.getColumns()
+	if s == nil || s.conn == nil || colInfo == nil {
+		return errors.New("store is nil")
+	}
+
+	if s.readOnly {
+		return errors.New("cannot perform update with RO connection")
+	}
+
+	if !colInfo.isLogstore {
+		return errors.New("this table type does not support adding values by timestamp")
+	}
+
+	insertSql := fmt.Sprintf(
+		`INSERT INTO %s (%s, %s) VALUES (?, ?)`,
```

Comment:
For timestamps, should one of this be a number, not a string? Might not matter, since sqlite everything is a string...

Reply:
this is just formatting the statement with our allowed table and column names (as strings), the Exec call after should bind the timestamp itself correctly (at least it has been in testing)